src/codegen/codegen.ts

```diff
@@ -82,17 +86,17 @@ export function generateDataFileDeclarations(files: DataFile[]): string {
   const fileKeyValuePairs = files
     .map(({ name }) => {
       const displayName = getFileNameWithoutExtension(name)
-      const isCsv = name.toLowerCase().endsWith('csv')
+      const isCSV = name.toLowerCase().endsWith('csv')
+
+      if (isCSV) {
+        return `
+        "${displayName}": await csv.parse(await fs.open('../Data/${name}'), { asObjects: true })`
```

Comment:
This is quite neat IMO. I'll ask someone from the core team if there are plans to build something like this for JSON files